test/conftest.py

```diff
@@ -0,0 +1,19 @@
+from ocrd.resolver import Resolver
+from ocrd_utils import pushd_popd
+from pytest import fixture
+
+from test.assets import assets as assets
+
+METS_KANT_BINARIZED = assets.url_of('kant_aufklaerung_1784-binarized/data/mets.xml')
+METS_HEROLD_SMALL = assets.url_of('SBB0000F29300010000/data/mets_one_file.xml')
+
+@fixture
+def workspace_kant_binarized():
```

Comment:
In removing test/base.py, you also removed setting up the logging system. This leaves me with **no** logs now.